proto/ostracon/abci/types.proto

```diff
@@ -224,6 +224,9 @@ message ResponseCheckTx {
   repeated Event events     = 7
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   string codespace = 8;
+  string         sender        = 9;  // MEMO: not used, just reservation for mempool_error
+  int64          priority      = 10; // MEMO: not used, just reservation for mempool_error
+  string         mempool_error = 11;
```

Comment:
How about leaving this comment here:

* https://github.com/tendermint/tendermint/blob/v0.34.24/proto/tendermint/abci/types.proto#L217-L219

```
  // mempool_error is set by Ostracon.
  // ABCI applictions creating a ResponseCheckTX should not set mempool_error.
```